website/addons/base/views.py

```diff
@@ -715,7 +715,7 @@ def addon_view_file(auth, node, file_node, version):
         'extra': version.metadata.get('extra', {}),
         'size': version.size if version.size is not None else 9966699,
         'private': getattr(node.get_addon(file_node.provider), 'is_private', False),
-        'file_tags': [tag._id for tag in file_node.tags],
+        'file_tags': file_node.tags.all().values_list('id', flat=True),
```

Comment:
Same thing about system tags here. And also, `id` is going to be an integer primary key, not the guid string. Just wanted to make sure that the caller was accounting for that difference.